client/src/i18n/en/patient_records.json

```diff
@@ -58,6 +58,18 @@
       "IS_NOT_NEW_CASE":"Old Case",
       "IS_REFERED":"Is the patient a referral ?"
     },
+    "REPORT":{
+      "VISITS":"Patients Visits Report",
+      "DESCRIPTION":"This report shows visits stats by service",
```

Comment:
Let's make this say what the French says - "This report shows patient visitation statistics by service."